.maestro/subflows/repay.yaml

```
@@ -1,19 +1,27 @@
 appId: ${APP_ID ?? "app.exactly"}
 ---
-- scrollUntilVisible: { element: Repay }
+- scrollUntilVisible: { element: Upcoming payments }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)